sequins.go

```diff
@@ -132,14 +132,26 @@ func (s *sequins) initCluster() error {
 	if err != nil {
 		return err
 	}
-
+	ipAddresses, err := net.LookupHost(hostname)
+	ip := ""
+	if err	!= nil {
+		return err
+	}
+	if len(ipAddresses) < 1 {
+		ip = hostname
+	} else {
+		ip = ipAddresses[0]
```

Comment:
Hmm. I suspect that in the event there are multiple IPs we want to just use the hostname?